src/twisted/conch/test/test_connection.py

```diff
@@ -241,9 +242,16 @@ def test_serviceStopped(self):
         self.conn.openChannel(channel2)
         self.conn.ssh_CHANNEL_OPEN_CONFIRMATION(b'\x00\x00\x00\x00' * 4)
         self.assertTrue(channel1.gotOpen)
+        self.assertFalse(channel1.gotClosed)
```

Comment:
Did you add these to make sure your changes didn't incorrectly close a channel?

Reply:
Yes, but also just to make the tests more comprehensive. We know what that attribute should be at that point; might as well add the assertion.